### DISCUSSION TITLE
GitHub app bot seems to be stuck

### DISCUSSION THREAD

**Discussion by avioletdev** [renovatebot/renovate#22536] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I got renovate bot added to my repo

It created the config PR.

I edited the config and an error was displayed - I had already run the renovate-config-validator on the config so I thought it would be OK.

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    "schedule:earlyMondays",
    "group:allNonMajor",
    "timezone:Europe/London"
  ],
  "prCreation": "immediate",
  "baseBranches": [
    "develop"
  ],
  "branchPrefix": "OAP-",
  "branchTopic": "reno",
  "packageRules": [
    {
      "matchUpdateTypes": [
        "minor",
        "patch"
      ],
      "enabled": true
    },
    {
      "matchUpdateTypes": [
        "major"
      ],
      "enabled": false
    }
  ]
}
```

<img width="949" alt="Screenshot 2023-06-01 at 09 58 55" src="https://github.com/renovatebot/renovate/assets/16456015/71a5c12f-9204-439d-aeb0-457e0a2ab081">

I realised my mistake and moved the timezone out of the extends

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    "schedule:earlyMondays",
    "group:allNonMajor",
  ],
  "timezone": "Europe/London",
  "prCreation": "immediate",
  "baseBranches": [
    "develop"
  ],
  "branchPrefix": "OAP-",
  "branchTopic": "reno",
  "packageRules": [
    {
      "matchUpdateTypes": [
        "minor",
        "patch"
      ],
      "enabled": true
    },
    {
      "matchUpdateTypes": [
        "major"
      ],
      "enabled": false
    }
  ]
}
```
and then pushed that up.

Nothing happened for a couple of hours so I removed timezone from the config as I noticed UTC is the default anyway.

Nothing happened over night so I just pushed the exact same config.

Its showing daily pending as the current status - https://developer.mend.io/github/sumup/orderandpay-backend/-/job/0188744a-4e07-70e9-b230-19f645f2d0d0

I am not sure if a minimal example makes sense in this case, I have provided the config and I am still at the onboarding stage :)

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
If you make another change to your onboarding branch config, does that trigger Renovate to run again, or is the job still pending? Ideally any push to Renovate config in an onboarding branch should trigger Renovate to run, so it's a backend bug if that's not the case.

Cc @nabeelsaabna

**Reply by nabeelsaabna** [reply]
checking

**Reply by avioletdev** [reply]
I have made a couple of changes. One using GitHub to edit the file, one from my IDE. Neither triggered a change.

**Reply by nabeelsaabna** [reply]
Hi @Beancow, problem should be fixed now

**Reply by avioletdev** [reply]
It's there now! Thank you so much!

**Reply by avioletdev** [reply]
@nabeelsaabna I made another change 30 ish minutes ago and its not updated. How long should I wait between config changes to see the effect on the PR? :)

**Reply by avioletdev** [reply]
The logs show this as pending again and its been a while :) 

https://developer.mend.io/github/sumup/orderandpay-backend/-/job/018876d4-5f82-7651-a648-8a591f054a3c

**Reply by nabeelsaabna** [reply]
I see you have a pending `hourly` job which gets added automatically by the system (just like the previous `daily` job),
but for some reason webhook events are not received or mishandled.
looking into it.. was there any specific action made to the repo since is was added to the Renovate app ?

**Reply by avioletdev** [reply]
Thank you for all your help! It's been like this since we added it. To begin with edits to the renovate.json were completely ignored. I thought removing the permissions and re-adding them might trigger the bot, so I did that a couple of times. There is another stuck pending job at the moment, I missed a comma on my config edit. 

Is there a way for us to trigger a retry? :) 

Does it look up the repo each time? We have an issue in other services where the lookup sometimes fails due to the number of repos we have.

**Comment by rarkins** [comment, marked as answer]
Hi, we've found a bug where commits to onboarding branches are being ignored, meaning Renovate does not immediately run again. In addition, daily jobs, which are used to check onboarding repos, were not getting prioritized enough. We are adjusting resources so that daily jobs should get executed once per day as intended, and the commit webhook bug should be fixed on Sunday this week.
